Packs/DemistoRESTAPI/CONTRIBUTORS.json

```diff
@@ -0,0 +1,3 @@
+[
```

Comment:
Please move this file to `CommunityCommonScripts`.

Reply:
I've removed it, since it doesn't seam common practice in the community pack